doc/source/how_to_install.md

```diff
@@ -28,9 +28,18 @@ Updating packages with `pip` (and sometimes `mamba`) can break your installation
 
 ## Installing for contributors
 
+### With ``mamba``
+
 ```bash
 git clone https://github.com/GlacioHack/xdem.git
 mamba env create -f xdem/dev-environment.yml
 ```
 
+### With ``pip``
+
+```bash
+git clone https://github.com/GlacioHack/xdem.git
+make install
+```
+
 After installing, you can check that everything is working by running the tests: `pytest`.
```

Comment:
maybe separate a way with conda and a way with Makefile (as commented, to be discussed in next meeting for future way...)